packages/appium/docs/en/quickstart/test-dotnet.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Write a Test (dotnet)
+---
+
+The [Appium dotnet Client](https://github.com/appium/dotnet-client/) is
+an official Appium client in C#. This driver is an extension of the Selenium C# client. It has all the functionalities of the regular driver, but add Appium-specific methods on top of this. The driver is not available under Nuget, since some of the dependencies do not have a signed version. With that said, you are more than welcome to download the code and build a signed version yourself. For the purposes of this quickstart we will proceed to clone the project code.

```

Comment:
What do you mean by the driver is not available under NuGet?

Reply:
I literally copy-pasted this disclaimer from https://github.com/appium/dotnet-client/

![image](https://github.com/appium/appium/assets/952392/0bf78e67-eb66-49f3-808f-c606749f4218)

A quick search in Nuget got me this package (https://www.nuget.org/packages/Appium.WebDriver/5.0.0-rc.1) which seems like the official one and I do see you as contributor. I will give it a try and update this PR. Should we also update the dotnet-client one?

Reply:
Not sure about this disclaimer, maybe @laolubenson recall what it's about.
Anyway, the version you sent it's the official release of the project. I still need to release RC2 in the future, maybe even this week.

Reply:
@Dor-bl  fixing doc, adding link to the nuget version. Confirming it continue to works.

![image](https://github.com/appium/appium/assets/952392/c7bf5c1f-cfa6-4be6-8064-f7509b1ad830)

Let me know if you need help with rc2, coding or testing.

Reply:
Cool, thanks for the effort.